source/linux/How_to_Guides/FAQ/How_to_Check_Device_Tree_Info.rst

```diff
@@ -9,81 +9,152 @@ using the entries in /proc/device-tree.
+
+		# Finding symbols for R5Fs
+		root@<machine>:~# ls /proc/device-tree/__symbols__/main_r5fss*
+		/proc/device-tree/__symbols__/main_r5fss0                          /proc/device-tree/__symbols__/main_r5fss0_core1_dma_memory_region  /proc/device-tree/__symbols__/main_r5fss1_core0_memory_region
+		/proc/device-tree/__symbols__/main_r5fss0_core0                    /proc/device-tree/__symbols__/main_r5fss0_core1_memory_region      /proc/device-tree/__symbols__/main_r5fss1_core1
+		/proc/device-tree/__symbols__/main_r5fss0_core0_dma_memory_region  /proc/device-tree/__symbols__/main_r5fss1                          /proc/device-tree/__symbols__/main_r5fss1_core1_dma_memory_region
+		/proc/device-tree/__symbols__/main_r5fss0_core0_memory_region      /proc/device-tree/__symbols__/main_r5fss1_core0                    /proc/device-tree/__symbols__/main_r5fss1_core1_memory_region
+		/proc/device-tree/__symbols__/main_r5fss0_core1                    /proc/device-tree/__symbols__/main_r5fss1_core0_dma_memory_region
+
+		# Finding location from the symbols
+		root@<machine>:~# cat /proc/device-tree/__symbols__/main_r5fss0_core0_memory_region
+		/reserved-memory/r5f-memory@a2100000
+
+		root@<machine>:~# cat /proc/device-tree/__symbols__/main_r5fss0_core0_dma_memory_region
+		/reserved-memory/r5f-dma-memory@a2000000
+
+		# Checking the values
+		root@<machine>:~# hexdump -C /proc/device-tree/reserved-memory/r5f-memory@a2100000/reg
+		00000000: 00000000 a2100000 00000000 01<address>0  ................
+
+		root@<machine>:~# hexdump -C /proc/device-tree/reserved-memory/r5f-dma-memory@a2000000/reg
+		00000000: 00000000 a2000000 00000000 00100000  ................
+
+.. ifconfig:: CONFIG_part_variant in ('AM62X', 'AM62PX', 'AM62AX', 'AM64X', 'AM65X')
+
+	.. code-block:: console
+
+		# Finding symbols for remote cores
+		root@<machine>:~# ls /proc/device-tree/__symbols__/ | grep "r5f"; ls /proc/device-tree/__symbols__/ | grep "m4f"; ls /proc/device-tree/__symbols__/ | grep "c7x";
+		wkup_r5fss0
+		wkup_r5fss0_core0
+		wkup_r5fss0_core0_dma_memory_region
+		wkup_r5fss0_core0_memory_region
+		mcu_m4fss
+		mcu_m4fss_dma_memory_region
+		mcu_m4fss_memory_region
+
+		# Finding location from the symbols
+		root@<machine>:~# cat /proc/device-tree/__symbols__/mcu_m4fss_memory_region
+		/reserved-memory/m4f-memory@9cc00000
+
+		root@<machine>:~# cat /proc/device-tree/__symbols__/mcu_m4fss_dma_memory_region
+		/reserved-memory/m4f-dma-memory@9cb00000
+
+		# Checking the values
+		root@<machine>:~# hexdump -C /proc/device-tree/reserved-memory/m4f-memory@9cc00000/reg
+		00000000  00 00 00 00 9c c0 00 00  00 00 00 00 00 e0 00 00  |................|
+		00000010
+
+		root@<machine>:~# hexdump -C /proc/device-tree/reserved-memory/m4f-dma-memory@9cb00000/reg
+		00000000  00 00 00 00 9c b0 00 00  00 00 00 00 00 10 00 00  |................|
+		00000010
+
+.. warning::
+
+	**IMPORTANT**: Before tweaking the memory for remote processors, please verify that the memory
```

Comment:
No need to use the `**IMPORTANT**` tag. See the following for eye catching notes:

https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#admonitions-messages-and-warnings